ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.collection.tsx

```diff
@@ -4,13 +4,16 @@ import { hot } from 'react-hot-loader'
 import MarionetteRegionContainer from '../../react-component/container/marionette-region-container'
 import styled from '../../react-component/styles/styled-components'
 
+const store = require('../../js/store.js')
```

Comment:
We shouldn't be pulling in the store directly, we should be using the reference that's passed to this component.